Modules/Core/Common/include/itkImageRegion.h

```diff
@@ -378,6 +413,31 @@ std::ostream &
 operator<<(std::ostream & os, const ImageRegion<VImageDimension> & region);
 } // end namespace itk
 
+
+namespace std
+{
+
+/** `std::tuple_size` specialization, needed for ImageRegion to support C++ structured binding.
+ *
+ * Example, using structured binding to retrieve the index and size of a region:
+   \code
+    auto [index, size] = image.GetRequestedRegion();
+   \endcode
+ */
+template <unsigned int VImageDimension>
+struct tuple_size<itk::ImageRegion<VImageDimension>> : std::integral_constant<size_t, 2>
```

Comment:
I wonder if these `std::` are needed inside `namespace std`?

Reply:
@issakomi  Good catch 👍 These particular `std::` inside namespace `std` are not needed, and I don't even like them  😺 Will remove them with the next force-push